### PR TITLE
docs: translate page `prefetchDNS`

### LINKED ISSUE #692: `prefetchDNS`
Page: https://id.react.dev/reference/react-dom/prefetchDNS

### PR DESCRIPTION
**Pull request by andriawan**
Closes #692

## Description

Translate the `prefetchDNS` page.
Page URL: https://id.react.dev/reference/react-dom/prefetchDNS

### CODE REVIEW

**Comment by andriawan** [comment]
ready to be reviewed @resir014

File: src/content/reference/react-dom/prefetchDNS.md [hunk 1]
```diff
@@ -77,9 +77,9 @@ function AppRoot() {
 }
 ```
 
-### Prefetching DNS in an event handler {/*prefetching-dns-in-an-event-handler*/}
+### Prefetching DNS pada event handler {/*prefetching-dns-in-an-event-handler*/}
 
-Call `prefetchDNS` in an event handler before transitioning to a page or state where external resources will be needed. This gets the process started earlier than if you call it during the rendering of the new page or state.
+Panggil `prefetchDNS` di dalam event handler sebelum bertransisi ke halaman atau state yang membutuhkan sumber daya eksternal. Hal ini akan memulai proses lebih awal dibandingkan jika Anda memanggilnya saat merender halaman atau state baru.
```
Comment (r17x) [inline src/content/reference/react-dom/prefetchDNS.md:82]:
```suggestion
Panggil `prefetchDNS` di dalam *event handler* sebelum bertransisi ke halaman atau *state*yang membutuhkan sumber daya eksternal. Hal ini akan memulai proses lebih awal dibandingkan jika Anda memanggilnya saat merender halaman atau *state* baru.
```

File: src/content/reference/react-dom/prefetchDNS.md [hunk 2]
```diff
@@ -39,34 +39,34 @@ function AppRoot() {
...
 ---
 
-## Usage {/*usage*/}
+## Penggunaan {/*usage*/}
 
-### Prefetching DNS when rendering {/*prefetching-dns-when-rendering*/}
+### Melakukan prefetching DNS saat merender {/*prefetching-dns-when-rendering*/}
 
-Call `prefetchDNS` when rendering a component if you know that its children will load external resources from that host.
+Panggil `prefetchDNS` saat merender komponen jika Anda mengetahui bahwa anak komponen akan memuat sumber daya eksternal dari host tersebut.
```
Comment (r17x) [inline src/content/reference/react-dom/prefetchDNS.md:69]:
```suggestion
Panggil `prefetchDNS` saat me-*render* komponen jika Anda mengetahui bahwa anak komponen akan memuat sumber daya eksternal dari host tersebut.
```

Comment (r17x) [inline src/content/reference/react-dom/prefetchDNS.md:69]:
```suggestion
Panggil `prefetchDNS` saat me-*render* komponen jika Anda mengetahui bahwa anak komponen akan memuat sumber daya eksternal dari host tersebut.
```

File: src/content/reference/react-dom/prefetchDNS.md [hunk 3]
```diff
@@ -39,34 +39,34 @@ function AppRoot() {
...
-* If you know the specific resources you'll need, you can call [other functions](/reference/react-dom/#resource-preloading-apis) instead that will start loading the resources right away.
-* There is no benefit to prefetching the same server the webpage itself is hosted from because it's already been looked up by the time the hint would be given.
-* Compared with [`preconnect`](/reference/react-dom/preconnect), `prefetchDNS` may be better if you are speculatively connecting to a large number of domains, in which case the overhead of preconnections might outweigh the benefit.
+* Beberapa panggilan ke `prefetchDNS` dengan server yang sama memiliki efek yang sama dengan panggilan tunggal.
+* Di browser, Anda dapat memanggil `prefetchDNS` dalam situasi apa pun: saat merender komponen, di Effect, di event handler, dan sebagainya.
+* Dalam rendering sisi server atau saat merender Komponen Server, `prefetchDNS` hanya memiliki efek jika Anda memanggilnya saat merender komponen atau dalam konteks asinkronisasi yang berasal dari rendering komponen. Pemanggilan lainnya akan diabaikan.
```
Comment (r17x) [inline src/content/reference/react-dom/prefetchDNS.md:58]:
```suggestion
* Dalam rendering sisi server atau saat me-*render* Komponen Server, `prefetchDNS` hanya memiliki efek jika Anda memanggilnya saat me-*render* komponen atau dalam konteks asinkronisasi yang berasal dari *rendering* komponen. Pemanggilan lainnya akan diabaikan.
```

File: src/content/reference/react-dom/prefetchDNS.md [hunk 4]
```diff
@@ -39,34 +39,34 @@ function AppRoot() {
...
-* In server-side rendering or when rendering Server Components, `prefetchDNS` only has an effect if you call it while rendering a component or in an async context originating from rendering a component. Any other calls will be ignored.
-* If you know the specific resources you'll need, you can call [other functions](/reference/react-dom/#resource-preloading-apis) instead that will start loading the resources right away.
-* There is no benefit to prefetching the same server the webpage itself is hosted from because it's already been looked up by the time the hint would be given.
-* Compared with [`preconnect`](/reference/react-dom/preconnect), `prefetchDNS` may be better if you are speculatively connecting to a large number of domains, in which case the overhead of preconnections might outweigh the benefit.
+* Beberapa panggilan ke `prefetchDNS` dengan server yang sama memiliki efek yang sama dengan panggilan tunggal.
+* Di browser, Anda dapat memanggil `prefetchDNS` dalam situasi apa pun: saat merender komponen, di Effect, di event handler, dan sebagainya.
```
Comment (r17x) [inline src/content/reference/react-dom/prefetchDNS.md:57]:
```suggestion
* Di browser, Anda dapat memanggil `prefetchDNS` dalam situasi apa pun: saat me-*render* komponen, di Effect, di *event handler*, dan sebagainya.
```

**Review by r17x — CHANGES_REQUESTED** [review]

**Review by r17x — APPROVED** [review]

File: src/content/reference/react-dom/prefetchDNS.md [hunk 5]
```diff
@@ -39,34 +39,34 @@ function AppRoot() {
...
+* Tidak ada manfaatnya untuk mengambil sumber daya dari server yang sama dengan tempat halaman web dihosting karena sumber daya tersebut sudah dicari pada saat petunjuk diberikan.
+* Dibandingkan dengan [`preconnect`](/reference/react-dom/preconnect), `prefetchDNS` mungkin lebih baik jika Anda secara spekulatif menyambungkan ke sejumlah besar domain, yang mana dalam hal ini biaya tambahan untuk melakukan prapenghubungan bisa jadi lebih besar daripada manfaatnya.
 
 ---
 
-## Usage {/*usage*/}
+## Penggunaan {/*usage*/}
 
-### Prefetching DNS when rendering {/*prefetching-dns-when-rendering*/}
+### Melakukan prefetching DNS saat merender {/*prefetching-dns-when-rendering*/}
```
Comment (r17x) [inline src/content/reference/react-dom/prefetchDNS.md:67]: Minor `merender` -> `me-render`

**Review by r17x — CHANGES_REQUESTED** [review]

File: src/content/reference/react-dom/prefetchDNS.md (hunk 5 above)
Comment (andriawan) [reply src/content/reference/react-dom/prefetchDNS.md]: done

**Review by r17x — APPROVED** [review]

**Review by resir014 — APPROVED** [review]

**Pull request merged**